spot_driver/spot_driver/spot_ros2.py

```diff
@@ -2825,6 +2838,22 @@ def handle_set_gripper_camera_parameters(
 
         return response
 
+    def handle_override_grasp_or_carry(
+        self,
+        request: OverrideGraspOrCarry.Request,
+        response: OverrideGraspOrCarry.Response,
+    ) -> OverrideGraspOrCarry.Response:
+        response = OverrideGraspOrCarry.Response()
+        if self.spot_wrapper is None or not self.spot_wrapper.has_arm():
+            response.success = False
+            return response
+        grasp_override = request.grasp_override.value
+        carry_override = request.carry_override.value
+        response.success, response.message = self.spot_wrapper.spot_arm.override_grasp_or_carry(
+            grasp_override, carry_override
+        )
```

Comment:
@jbarry-bdai nit^2: all this could be written in fewer lines:
```suggestion
        response.success, response.message = self.spot_wrapper.spot_arm.override_grasp_or_carry(
            request.grasp_override.value, request.carry_override.value
        )
```